envoy/upstream/outlier_detection.h

```diff
@@ -110,6 +210,22 @@ class DetectorHostMonitor {
    * and LocalOrigin type returns success rate for local origin errors.
    */
   virtual double successRate(SuccessRateMonitorType type) const PURE;
+
+  /* Returns name of failed extension monitor.
+     Returns empty string if failure was not in extensions.
+  */
+  virtual std::string getFailedExtensionMonitorName() const PURE;
+
+  /* Returns extra info which extension monitor wants to add to
+     event logger.
+     Returns empty string if failure was not in extensions.
+  */
+  virtual std::string getFailedExtensionMonitorExtraInfo() const PURE;
+
+  /* Returns ejection enforcing parameter configured in extension
+     monitor which failed.
+  */
+  virtual uint32_t getFailedExtensionMonitorEnforce() const PURE;
```

Comment:
Please check the style of these comments.

Reply:
Corrected. Thanks.